templates/zerver/app/settings_overlay.html

```diff
@@ -140,11 +140,13 @@ <h1>{{ _('Settings') }}<span class="section"></span></h1>
                     <div class="text">{{ _('Custom profile fields') }}</div>
                 </li>
                 {% endif %}
-                {% if is_admin %}
+                {% if not is_guest %}
                 <li tabindex="0" data-section="invites-list-admin">
-                    <i class="icon fa fa-user" aria-hidden="true"></i>
+                    <i class="icon fa fa-user-plus" aria-hidden="true"></i>
```

Comment:
Can you move this line to its own commit, changing the icon?  It's an independent conceptual change from the rest, and it's good practice to use a separate commit for this sort of thing.

Reply:
ok